### DISCUSSION TITLE
Not Working My github-readme-stats

### DISCUSSION THREAD

**Discussion by yumble** [anuraghazra/github-readme-stats#3428] [Q&A] [answered]
My private repository commits are not reflected
I tried using the following address, but it didn't work
-> https://github-readme-stats-yumbles-projects.vercel.app/api?username=yumble&show_icons=true&theme=radical&count_private=true&include_all_commits=true
-> https://github-readme-stats.vercel.app/api?username=yumble&include_all_commits=true

To be precise, it works, but the commitS of a specific repository are not reflected.
It should be over 1000, but it looks like 175, 226.

Why can't some repositories do when you've done over 1000 commitments this year?

**Comment by qwerty541** [comment, marked as answer]
Hey, @yumble! If you deployed your own Vercel instance using your own PAT (Personal Access Token (GitHub API key)) and private contributions still not appear at your stats card, the reason of your problem can be incorrect permissions of your PAT. Make sure that you enabled `repo` and `user` permissions.

![image](https://github.com/anuraghazra/github-readme-stats/assets/53787217/9c94bb06-a59d-402e-8014-5f821046b630)

![image](https://github.com/anuraghazra/github-readme-stats/assets/53787217/c835b969-9850-4cc9-82ed-b9207214b47f)

**Reply by yumble** [reply]
Oh,, Thank you very much..!! It's working now ☺️
